src/plugins/intel_cpu/tests/unit/ngraph_transformations/snippets/enforce_precision.cpp

```diff
@@ -0,0 +1,391 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
I meant the path `unit/snippets_transformations/enforce_precision.cpp` since we have the directories `ngraph_transformations` and `snippets_transformations` in plugin

Reply:
> I meant the path `unit/snippets_transformations/enforce_precision.cpp` since we have the directories `ngraph_transformations` and `snippets_transformations` in plugin

done